client/context/query.go

```diff
@@ -126,6 +135,7 @@ func (ctx CLIContext) EnsureAccountExistsFromAddr(addr sdk.AccAddress) error {
 
 // queryAccount queries an account using custom query endpoint of auth module
 // returns an error if result is `null` otherwise account data
+// returns the height the account was queried at
```

Comment:
This isn't true :)

Reply:
how so?

Reply:
`queryAccount` doesn't return a height @jackzampolin